joern-cli/frontends/jssrc2cpg/src/main/scala/io/joern/jssrc2cpg/utils/Environment.scala

```diff
@@ -5,8 +5,9 @@ import java.nio.file.Paths
 
 object Environment {
 
-  val IS_MAC: Boolean   = scala.util.Properties.isMac
-  val IS_LINUX: Boolean = scala.util.Properties.isLinux
+  val IS_MAC: Boolean     = scala.util.Properties.isMac
```

Comment:
if we change this to `IS_MAC_X86` (or similar) it'd be more specific, and the order of the conditional in `AstGenRUnner` wouldn't matter any more... just a thought

Reply:
Yes, right. I changed that to something more explicit. Please have a look :-)